lib/registries/addon/components/registries-license-picker/template.hbs

```diff
@@ -48,6 +48,7 @@
                             {{t (concat 'app_components.license_picker.fields.' key)}}
                             <Input
                                 data-test-required-field={{key}}
+                                aria-required='true'
```

Comment:
This one needed the `required=true` since it does not have the red asterisk, but if your [PR](https://github.com/CenterForOpenScience/ember-osf-web/pull/965) takes care of it, then it shouldn't be needed